packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/ios/Flutter/production.xcconfig

```diff
@@ -0,0 +1,5 @@
+#include "Generated.xcconfig"
+
+FLUTTER_TARGET=lib/main.dart
+ENVIRONMENT_PLIST = production.plist
```

Comment:
development.plist, staging.plist, production.plist are not visible (indlucded in the Xcode project)